data_handler.py

```diff
@@ -21,6 +21,9 @@ class BaseDataHandler:
     def __init__(self, *args, **kwargs):
         self._sample_size = kwargs.pop('sample', 0)
         self._skip_scrape = kwargs.pop('no_scrape', False)
+        self._outputter = kwargs.pop('outputter')
+
+        self.cvr_handler = get_cvr_handler()
```

Comment:
Is this intentional? The same statement is 2 lines further down.

Reply:
Definitely a mistake..